packages/tools/src/eventListeners/annotations/contourSegmentation/areContoursFromSameSegmentIndex.ts

```diff
@@ -0,0 +1,15 @@
+import { ContourSegmentationAnnotation } from '../../../types/ContourSegmentationAnnotation';
+
+export default function areContoursFromSameSegmentIndex(
+  firstAnnotation: ContourSegmentationAnnotation,
+  secondAnnotation: ContourSegmentationAnnotation
+) {
+  const { segmentation: firstSegmentation } = firstAnnotation.data;
+  const { segmentation: secondSegmentation } = secondAnnotation.data;
+
+  return (
```

Comment:
You should also chexck firstSegmentation.segmentationId is the same as the second one, no?